modules/packages/BLAS.chpl

```diff
@@ -973,7 +973,7 @@ module BLAS {
     TODO -- Get banded array routines working
   */
   proc hbmv(A: [?Adom] ?eltType, X: [?vDom] eltType, ref Y: [vDom] eltType,
-            ref alpha: eltType, ref beta: eltType,
+            alpha: eltType, const ref beta: eltType,
```

Comment:
Shouldn't the `ref` intent on `beta` be removed rather than changed to `const ref`? Same for `hemv`, `hpmv`

Reply:
Yes…  yes it should, thanks for catching that!  🤦

Reply:
Resolved by https://github.com/chapel-lang/chapel/pull/24898/commits/34834a6a502ef66823a45a1d0fb298a8e3a8eb2e, thanks again for catching that!  (I did a number of sanity checks, but apparently that was not one of them…  And I apparently either never read the diff or my eyes glazed over too quickly).